frontendFile/src/main/grammar/Machete.flex

```diff
@@ -0,0 +1,58 @@
+package com.virtuslab.gitmachete.frontend.file.grammar;
+
+import com.intellij.lexer.FlexLexer;
+import com.intellij.psi.tree.IElementType;
+import com.intellij.psi.TokenType;
+
+%%
+
+%class MacheteGeneratedLexer
+%implements FlexLexer
+%unicode
+%function advance
+%type IElementType
+%eof{  return;
+%eof}
+
+
+// End Of Line
+EOL=[\r\n]+
+// For sake of distinguishablility WHITESPACE and INDENTATION are two separated entries
+// even if they are the same for now
+WHITESPACE=[\ \t]+
+INDENTATION=[\ \t]+
+// Probably all characters allowed in branch name
+// (https://mirrors.edge.kernel.org/pub/software/scm/git/docs/git-check-ref-format.html)
+// \w is standard regex abbreviation of [a-zA-Z0-9_]
+NAME_WITHOUT_SLASH=[\w\-.!@#$%&()+={}\];'\",.<>|`]+
+// In addition, names can contain / (slash)
+NAME_WITH_SLASH=[\w\-.!@#$%&()+={}\];'\",.<>|`\/]+
+SLASH="\/"
+ANNOTATION=[^\r\n]+
```

Comment:
What if a different newline encoding is used? :thinking: Did you test it on Windows?

Reply:
No, I didn't test this on Windows, but this regex should catch annotations regardless new line type.
It is simply all characters except `\n` and `\r` so either `\r` on Mac, `\n` on Linux and `\r\n` on Windows should not be catched.